src/lib/support/CodeUtils.h

```diff
@@ -245,12 +246,12 @@
  *  Example usage:
  *
  *  @code
- *    LogAndReturnErrorOnFailure(channel->SendMsg(msg), Module, "Failure message: %s", param);
+ *    ReturnErrorAndLogOnFailure(channel->SendMsg(msg), Module, "Failure message: %s", param);
  *  @endcode
  *
  *  @param[in]  expr        A scalar expression to be evaluated against CHIP_NO_ERROR.
  */
-#define LogAndReturnErrorOnFailure(expr, MOD, MSG, ...)                                                                            \
+#define ReturnErrorAndLogOnFailure(expr, MOD, MSG, ...)                                                                            \
```

Comment:
Continuing discussion @bzbarsky-apple : I just noticed that `ReturnLogErrorOnFailure`, which you suggested we could combine, also logs `__FILE__` and `__LINE__` explicitly; do we really want that? i.e. the ChipError will already have that in debug builds, seems a bit much to also log file/line where we caught the error...

If you agree that's unnecessary, I can convert `ReturnLogErrorOnFailure` to use this